public/static/pathways/neoadjuvant_pathway.json

```diff
@@ -242,91 +182,14 @@
         "action": [
           {
             "type": "create",
-            "description": "Cyclophosphamide Medication request",
-            "resource": {
-              "resourceType": "MedicationRequest",
-              "medicationCodeableConcept": {
-                "coding": [
-                  {
-                    "system": "http://www.nlm.nih.gov/research/umls/rxnorm",
-                    "code": "1734919",
-                    "display": "Cyclophosphamide 1000 MG Injection"
-                  }
-                ],
-                "text": "Cyclophosphamide 1000 MG Injection"
-              }
-            }
-          },
-          {
-            "type": "create",
-            "description": "Doxorubicin Medication request",
-            "resource": {
-              "resourceType": "MedicationRequest",
-              "medicationCodeableConcept": {
-                "coding": [
-                  {
-                    "system": "http://www.nlm.nih.gov/research/umls/rxnorm",
-                    "code": "1790100",
-                    "display": "Doxorubicin"
-                  }
-                ],
-                "text": "Doxorubicin"
-              }
-            }
-          },
-          {
-            "type": "create",
-            "description": "Pertuzumab Medication request",
-            "resource": {
-              "resourceType": "MedicationRequest",
-              "medicationCodeableConcept": {
-                "coding": [
-                  {
-                    "system": "http://www.nlm.nih.gov/research/umls/rxnorm",
-                    "code": "1298948",
-                    "display": "pertuzumab"
-                  }
-                ],
-                "text": "pertuzumab"
-              }
-            }
-          },
-          {
-            "type": "create",
-            "description": "Trastuzumab Medication request",
-            "resource": {
-              "resourceType": "MedicationRequest",
-              "medicationCodeableConcept": {
-                "coding": [
-                  {
-                    "system": "http://www.nlm.nih.gov/research/umls/rxnorm",
-                    "code": "1922509",
-                    "display": "trastuzumab"
-                  }
-                ],
-                "text": "trastuzumab"
-              }
-            }
-          },
-          {
-            "type": "create",
-            "description": "Paclitaxel Medication request",
+            "description": "Chemo Care Plan",
```

Comment:
Can you update this one to say "Chemotherapy ACTHP Regimen"? Also since the Type field in the UI already says "care plan" also could you rename the other ones to say Regimen as well? That's less duplicative 
![image](https://user-images.githubusercontent.com/13512036/83526492-7f000e00-a4b4-11ea-8f1f-12f3f8905082.png)

Reply:
There is already a Chemotherapy ATCHP, are there meant to be 2?

Reply:
This state already has the label Chemotherapy ATCHP but the description is just generic. See the middle one stands out
![image](https://user-images.githubusercontent.com/13512036/83529255-19ae1c00-a4b8-11ea-8bb5-e7fa4a5a2a22.png)